api/management/commands/loaddata.py

```diff
@@ -0,0 +1,17 @@
+import os
+import csv
+from django.core.management.base import BaseCommand
+from api.models import Disease
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        path = os.path.join('data', 'diseases.tsv')
+        with open(path, 'r') as diseases_file:
+            diseases_rows = csv.reader(diseases_file, delimiter='\t')
+            # Skip header
+            next(diseases_rows)
+            for row in diseases_rows:
+                acronym = row[0]
+                name = row[1]
+                Disease.objects.create(acronym=acronym, name=name)
```

Comment:
Will these create functions work if the arguments are not the right type? Not relevant here, but I'm curious for elsewhere.